intake_elasticsearch/elasticsearch_table.py

```diff
@@ -60,15 +59,24 @@ def _get_schema(self):
                            npartitions=1,
                            extra_metadata=self.metadata)
 
-    def to_dask(self):
+    def to_dask(self, npartitions=1):
         """Make single-partition lazy dask data-frame"""
         import dask.dataframe as dd
         from dask import delayed
         self.discover()
-        part = delayed(self._get_partition(0))
-        return dd.from_delayed([part], meta=self.dtype)
+        parts = []
+        if npartitions == 1:
+            part = delayed(self._get_partition(0))
+            parts.append(part)
+        else:
+            for slice_id in range(npartitions):
+                self._dataframe = None
+                part = delayed(self._get_partition(0, slice_id=slice_id,
```

Comment:
should, I think, be delayed(self._get_partition)(args), otherwise, this is evaluated immediately, and only the result delayed, which is not useful. Indeed, I would do
```
dpart = delayed(self._get_partition)
for slice_id in range(npartitions):
    part = dpart(args)
    ...
```

Reply:
I think you are right. Missed that.